data_safe_haven/infrastructure/stacks/shm/firewall.py

```diff
@@ -373,6 +282,7 @@ def __init__(
                             source_addresses=[props.subnet_identity_servers_iprange],
                             target_fqdns=[
                                 "au.download.windowsupdate.com",
+                                #                            "{{storage.artifacts.accountName}}.blob.core.windows.net",
```

Comment:
I think we probably don't need this?

Reply:
Yep, was going to ask if that was absent because it wasn't necessary any more